.github/workflows/docker.yml

```diff
@@ -38,51 +39,27 @@ jobs:
             suffix: ""
 
     steps:
-      - uses: actions/checkout@v2
```

Comment:
Is there a reason why you moved these three actions after the login to ECR?

Reply:
Yes and no. While testing, the login to ECR failed a couple of times, and wanted to do it first to speed up the development.
I can change the order to the old one if necessary.